slices/rabbitmq-server.yaml

```diff
@@ -0,0 +1,114 @@
+package: rabbitmq-server
+
+essential:
+  - rabbitmq-server_copyright
+
+slices:
+  autocomplete:
+    contents:
+      /usr/lib/rabbitmq/autocomplete/*:
+
+  bins:
+    essential:
+      - base-files_base
+      - base-passwd_data
+      - bash_bins
+      - coreutils_bins
+      ## erlang* packages do not exist for i386
+      ## for all other archs, uncomment all those slices below
+      # - erlang-base_bins
+      # - erlang-inets_bins
+      # - erlang-os-mon_bins
+      - grep_bins
+      - libc-bin_locale
+      - libc-bin_nsswitch
+      - libtinfo6_libs
+      - login_bins
+      - netbase_default-hosts
+      - netbase_default-networks
+      - passwd_bins
+      - python3.12_standard
+      - rabbitmq-server_libs
+      - sed_bins
+      - util-linux_bins
+    contents:
+      /usr/bin/rabbitmqadmin:
+      /usr/lib/ocf/resource.d/rabbitmq/rabbitmq-server:
+      /usr/lib/rabbitmq/bin/rabbitmq*:
+      /usr/lib/rabbitmq/bin/vmware-rabbitmq:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-*/escript/rabbitmq-diagnostics:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-*/sbin/rabbitmq*:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-*/sbin/vmware-rabbitmq:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-plugins:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-queues:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-streams:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmq-upgrade:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/rabbitmqctl:
+        { mutable: true }
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/escript/vmware-rabbitmq:
+        { mutable: true }
+      /usr/sbin/rabbitmq*:
+      /var/lib/rabbitmq/:
+      /var/lib/rabbitmq/.erlang.cookie: { text: DUMMY, mode: 0400 }
+      /var/lib/rabbitmq/mnesia/: { mode: 0750 }
+      /var/log/rabbitmq/:
+    mutate: |
```

Comment:
hum, if all of those files are just copies, you can just use `{ copy: ... }` in the paths above

Reply:
Those are hardlinks.

Reply:
Noted, but that isn't relevant here since chisel doesn't support hardlinks anyway. Using a mutation won't help with that. For other slices with hardlinks we've currently worked around via copies.

In any case, if these are hardlinks then we definitely need a TODO comment so we fix them once hardlinks are supported.